src/libcrun/utils.c

```diff
@@ -1403,15 +1403,15 @@ getsubidrange (uid_t id, int is_uid, uint32_t *from, uint32_t *len)
 #define MIN(x, y) ((x) < (y) ? (x) : (y))
 
 size_t
-format_default_id_mapping (char **ret, uid_t container_id, uid_t host_id, int is_uid)
+format_default_id_mapping (char **ret, uid_t container_id, uid_t host_uid, uid_t host_id, int is_uid)
```

Comment:
Shouldn't this be (..., uid_t host_uid, uid_t host_gid,...)

Reply:
@rhatdan is this resolved I think this has same explanation as the above one.